package.json

```diff
@@ -102,6 +108,8 @@
     "postcss-loader": "^2.0.8",
     "raf": "^3.4.0",
     "react-dev-utils": "^5.0.0",
+    "react-intl-cra": "^0.3.3",
```

Comment:
Do we actually need this ?

Reply:
Yes, it is the module that extracts the messages from `<FormattedMessage />` `<FormattedHTMLMessage />`, `defineMessage()`